[Stream] Added Enable outputs section (#5725)
cloudflare/cloudflare-docs: content/stream/stream-live/simulcasting.md

}
```

## Enable outputs

> You can actively control when you broadcast video by enabling or disabling outputs added to a live input. When a live input is in use for streaming and has a disabled output added to it, the video is not restreamed to the output. 

Reviewer: What does it mean to "actively" control vs. control?

Can you take a stab at explaining _what and why_ someone would use this for? (thinking back to use cases we went over)

This explains technically what happens, but our docs should help people understand how they use the feature to achieve the outcome they're seeking. (ex: start a stream to prepare before starting to restream, turn off an output while continuing to stream, etc.)

Consider the title as well — what is the customer trying to achieve? Outputs is _our_ concept — what are they searching the docs to figure out how to do?

Author: @irvinebroque Added an example of how could use this feature and updated the header. Not sure if I love the new header title I added. Open to feedback if something else is clearer.

By default, all outputs are enabled.

1. From Live Inputs in the [Cloudflare dashboard](https://dash.cloudflare.com/?to=/:account/stream/inputs), select an input from the list.

> 2. Under **Outputs** > **Enabled**, set the toggle to enabled or disabled.

Reviewer: This is possible via the API, and that's a likely path for someone reading docs. This should link to the relevant API docs.

Reviewer 2: I was just about to comment this, and also I see other examples on this page showing example `curl` usage. The public API docs for this endpoint are [here](https://api.cloudflare.com/#stream-live-inputs-update-a-single-output-on-a-live-input) which should serve as a good reference.

Author: @irvinebroque @nnazo Ah, good catch for the API docs. I added a link to it without adding the curl commands but let me know if the command should be included.

}
```

> ## Enable live broadcasting

Reviewer: ```suggestion
## Control when you start and stop simulcasting
```

> You can control when you broadcast video by enabling or disabling outputs added to a live input. For example, if you are preparing to host a live stream, you can start the stream while the output is disabled. As soon as you are ready to officially go live, you can enable the output associate with the live input.

Reviewer: ```suggestion
You can enable and disable individual live outputs via the [API](https://api.cloudflare.com/#stream-live-inputs-update-a-single-output-on-a-live-input) or [Stream dashboard](https://dash.cloudflare.com/?to=/:account/stream/inputs), allowing you to:

- Start a live stream, but wait to start simulcasting to YouTube and Twitch until right before the content begins.
- Stop simulcasting before the live stream ends, to encourage viewers to transition from a third-party service like YouTube or Twitch to a direct live stream.
- Give your own users manual control over when they go live to specific simulcasting destinations.
```

> When a live input is in use for streaming and has a disabled output added to it, the video is not restreamed to the output. 

Reviewer: ```suggestion
When a live output is disabled, video is not simulcast to the live output, even when actively streaming to the corresponding live input.
```

By default, all outputs are enabled.

> 

Reviewer: Maybe add a small heading to denote that these are dashboard instructions, and a small screenshot as a visual cue.

Author: Added an H3 for this section but didn't add the screenshot because I think the header and first step make it clear it's the dashboard. If we learn this is confusing for users though, I can add in the screenshot.

Reviewer: Sounds good!

1. From Live Inputs in the [Cloudflare dashboard](https://dash.cloudflare.com/?to=/:account/stream/inputs), select an input from the list.
2. Under **Outputs** > **Enabled**, set the toggle to enabled or disabled.

> You can also use the Stream API to [Update a single output on a live input](https://api.cloudflare.com/#stream-live-inputs-update-a-single-output-on-a-live-input).

Reviewer: ```suggestion
```
Think we can bring this up to the top of the section and omit at the bottom.
